playlists/excel.yaml

```diff
@@ -828,12 +837,30 @@
   group: Preview APIs
   api_set:
     ExcelApi: '1.10'
-- id: excel-comment
-  name: Comments
-  fileName: comment.yaml
-  description: 'Adds, edits, and removes comments and comment replies.'
+- id: excel-comment-mentions
+  name: Comment mentions
+  fileName: comment-mentions.yaml
+  description: Mentions someone in a comment.
```

Comment:
Nit: in some entries, the value of `description` is enclosed in single quotes, in other entries (like this one) it is not. Does it matter?

Reply:
Sometimes the build script automatically adds single quotes, sometimes it doesn't. I'm not sure why, but both work.